app/app.rb

```diff
@@ -0,0 +1,102 @@
+require 'sinatra/base'
+require_relative './data_mapper_setup'
+require 'sinatra/flash'
+require 'mailgun'
+
+class Chitter < Sinatra::Base
+  set :views, proc {File.join(root,'..','/app/views')}
+  enable :sessions
+  register Sinatra::Flash
+  use Rack::MethodOverride
+  set :session_secret, 'super secret'
+
+  helpers do
+    def current_user
+      User.get(session[:user_id])
+    end
+
+    def send_email
+      RestClient::Request.execute(
+      url: "https://api:key-7af11fe0adc2a642e55d33744117e7d2"\
+      "@api.mailgun.net/v3/sandbox1aad2b9369f348018e3b435a1026927c.mailgun.org/messages",
+      :method => :post,
+      :payload => {
+      :from => 'Mailgun Sandbox <postmaster@sandbox1aad2b9369f348018e3b435a1026927c.mailgun.org>',
+      :sender => 'Mailgun Sandbox <postmaster@sandbox1aad2b9369f348018e3b435a1026927c.mailgun.org>',
+      :to => params[:email],
+      :subject => "Welcome to Chitter",
+      :text => "Welcome to Adrian's awesome Chitter site. " + params[:username] + ".Get Chittering!",
```

Comment:
Metrics/LineLength: Line is too long. [101/90]<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.